src/api/v1/statuses.ts

```diff
@@ -1010,23 +1111,39 @@ app.post(
       } satisfies NewPinnedPost)
       .returning();
     const fedCtx = federation.createContext(c.req.raw, undefined);
-    await fedCtx.sendActivity(
-      owner,
-      "followers",
-      new Add({
-        id: new URL(
-          `#add/${result[0].index}`,
-          fedCtx.getFeaturedUri(owner.handle),
-        ),
-        actor: new URL(owner.account.iri),
-        object: new URL(post.iri),
-        target: fedCtx.getFeaturedUri(owner.handle),
-      }),
-      {
-        preferSharedInbox: true,
-        excludeBaseUris: [new URL(c.req.url)],
-      },
-    );
+    const activity = new Add({
+      id: new URL(
+        `#add/${result[0].index}`,
+        fedCtx.getFeaturedUri(owner.handle),
+      ),
+      actor: new URL(owner.account.iri),
+      object: new URL(post.iri),
+      target: fedCtx.getFeaturedUri(owner.handle),
+    });
+    await fedCtx.sendActivity(owner, "followers", activity, {
+      preferSharedInbox: true,
+      excludeBaseUris: [new URL(c.req.url)],
+    });
+    if (!owner.account.protected) {
+      const acceptedRelays = await db.query.relays.findMany({
+        where: eq(relays.state, "accepted"),
+        with: {
+          relayServerActor: true,
+        },
+      });
+      await fedCtx.sendActivity(
+        owner,
+        acceptedRelays.map((relay) => ({
+          id: new URL(relay.relayServerActor.iri),
+          inboxId: new URL(relay.relayServerActor.inboxUrl),
+        })),
+        activity,
+        {
+          preferSharedInbox: true,
```

Comment:
We don't have to turn this on as we aren't providing recipients' shared inbox URLs here.